### PR TITLE
Added social share icons to seller page

### PR DESCRIPTION
**Pull request by danthaman44**
Social icons implemented via https://github.com/nygardk/react-share
Screen Shot 2020-05-07 at 12 53 52 AM
Screen Shot 2020-05-07 at 9 35 27 PM

### CODE REVIEW

File: src/components/OwnerPanel/OwnerPanel.tsx [hunk 1]
```diff
@@ -45,6 +53,9 @@ const OwnerPanel = (props: Props) => {
     return props.extraInfo[current] != null;
   });
 
+  const location = window.location.href;
+  const facebookQuote = 'Help raise money for ' + props.sellerName;
```
Comment (danthaman44) [inline src/components/OwnerPanel/OwnerPanel.tsx:57]: I just improvised this on the spot...any suggestions?

Comment (nanxiy) [reply src/components/OwnerPanel/OwnerPanel.tsx:57]: gonna slack you the icons we've been using

Comment (jtmckibb) [reply src/components/OwnerPanel/OwnerPanel.tsx:57]: Chinatown has been suffering from the effects of COVID. Help keep Chinatown alive by donating/buying a gift card! Check out <Insert Merchant Name>'s page:

idk something like that?

File: src/components/OwnerPanel/OwnerPanel.tsx [hunk 2]
```diff
@@ -170,13 +181,31 @@ const OwnerPanel = (props: Props) => {
...
+            </TwitterShareButton>
+          </div>
+
+          <div className={styles.socialIconContainer}>
+            <EmailShareButton url={location} className="share">
+              <EmailIcon size={48} round={true} />
+            </EmailShareButton>
+          </div>
+        </div>
+      }
```
Comment (bvillaroman) [inline src/components/OwnerPanel/OwnerPanel.tsx:209]: it looks like you're the most familiar with the new Owner Panel. So soon we should convert the styles to styled-components. In the next couple of prs you should see how i do styled components for components that apply for just that page. If we find ourselves reusing code then we can do another type of style of creating styled-components. But this is great!

Comment (danthaman44) [reply src/components/OwnerPanel/OwnerPanel.tsx:224]: Sounds good!

**Pull request merged**